projects/models.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import absolute_import, division, unicode_literals
+
+from django.db import models
+from django.utils.encoding import python_2_unicode_compatible
```

Comment:
There are some flake8 issues. You should be able to run `flake8 .` to see what they are. They are mostly unused imports.